### DISCUSSION TITLE
Local font for Text element

### DISCUSSION THREAD

**Discussion by marcelpi** [pmndrs/drei#898] [Q&A] [answered]
Hi there,

I'm trying to use a local font for a troika <Text> component. From what I see when specifying the font you need to put a url string as value in order to work.

<Text
    ...
    font='https://fonts.gstatic.com/font/some-font.woff'
/>

Is there a way I can point to a local font I've got in my project folder?
Something like this: 
<Text
    ...
    font='./public/assets/fonts/font.ttf'
/>

Or any other way really. Can it be done?

**Comment by mwmwmw** [comment, marked as answer]
This will depend on what build system you're using, but, if you want to use what is in the public folder, don't include `public` in the path.

```
<Text font='/assets/fonts/font.ttf' />
```

**Reply by marcelpi** [reply]
Thanks, it works. I was passing a .woff2 file which doesn't seem to be supported, besides the path error.
Cheers!
